tests/unit/test_svs.cpp

```diff
@@ -1,23 +1,22 @@
 #include "gtest/gtest.h"
 #include "VecSim/vec_sim.h"
 #include "unit_test_utils.h"
-#include "VecSim/algorithms/svs/svs.h"
-#include "cpu_features_macros.h"
 #include <cmath>
-#include <cpuid.h>
-#include <string>
 
-template <typename index_type_t>
-class SVSTest : public ::testing::Test {
-    bool _checkCPU() {
-        uint32_t eax, ebx, ecx, edx;
-        __cpuid(0, eax, ebx, ecx, edx);
-        std::string vendor_id = std::string((const char *)&ebx, 4) +
-                                std::string((const char *)&edx, 4) +
-                                std::string((const char *)&ecx, 4);
-        return (vendor_id == "GenuineIntel");
+#if HAVE_SVS
+#include "VecSim/algorithms/svs/svs.h"
+
+#define ASSERT_INDEX(index)                                                                        \
+    if (index == nullptr) {                                                                        \
+        if (std::get<1>(svs_details::isSVSQuantBitsSupported(TypeParam::get_quant_bits()))) {      \
```

Comment:
`std::get<1>(svs_details::isSVSQuantBitsSupported(TypeParam::get_quant_bits()))` always returns true currenly right? Can you explain the purpose of this assertion (now and in the future)?

Reply:
Added comment

Reply:
IIUC, lvq test should have been skipped in the CI (amd arch)
but they pass
`Test #1386: SVSTest.svs_reindexing_same_vector_different_id<SVSIndexType<(VecSimType)0,float,(VecSimSvsQuantBits)8>> ..................   Passed    0.01 sec`

what am i missing

Reply:
As we discussed already, SVS Index should fallback to non-LVQ implementation in case if LVQ is not supported on the platform - it is non-quantized fall back for now and the Scalar Global Quantization (intel/ScalableVectorSearch#97) in future.
This why tests are passed.